app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -525,4 +526,13 @@ public static boolean isLazyMediaDownloadComplete() {
                 .getBoolean(LAZY_MEDIA_DOWNLOAD_COMPLETE, false);
     }
 
+    public static boolean isFileOversizeWarningDisabled() {
+        if (CommCareApplication.instance().isConsumerApp()) {
```

Comment:
why are we overiding for consumer apps ?

Reply:
Ohh I kept it same as `isIncompleteFormsEnabled`. I thought we don't allow custom settings in consumer apps?

Reply:
I think thats only true for incomplete forms setting and not a general rule as all other custom settings doesn't have this check.